ortools/xpress/environment.cc

```diff
@@ -2203,6 +2203,14 @@ void LoadXpressFunctions(DynamicLibrary* xpress_dynamic_library) {
   xpress_dynamic_library->GetFunction(&XPRSbasiscondition,
                                       "XPRSbasiscondition");
   xpress_dynamic_library->GetFunction(&XPRSrefinemipsol, "XPRSrefinemipsol");
+
+  auto notFound = xpress_dynamic_library->FunctionsNotFound();
+  if (!notFound.empty()) {
+    return absl::NotFoundError(absl::StrCat("Could not find the following functions. [",
```

Comment:
maybe add a note to warn that the list is not exhaustive "eq (list is not exhaustive)"

Reply:
done !